test/integration/scheduler/predicates_test.go

```diff
@@ -54,6 +54,7 @@ func TestInterPodAffinity(t *testing.T) {
 		"zone":   "z11",
 	}
 	for _, node := range nodes {
+		// TODO: we could directly initialize the node with labels
```

Comment:
nit: `TODO(nodo)`

Reply:
also, TODOs should say what needs to be done.

TODO(nodo): Use PodWrapper to directly initialize node with labels.

Reply:
done 👍